### DISCUSSION TITLE
Templates Update to Primevue v4

### DISCUSSION THREAD

**Discussion by fotisskal** [primefaces/community#2594] [PrimeVue Templates] [answered]
Hello everyone.

Do you know by when, the templates eg. Atlantis, are estimated to get an update to Primevue v4? On website, it says Q3. So should we expect the v4 update by the end of September, earlier or later?

Thank you in advance!

**Comment by tugcekucukoglu** [comment, marked as answer]
It is already released for version 4. you could download from the PrimeStore.
